internal/cmd/argus/instance/create/create.go

```diff
@@ -64,6 +64,7 @@ func NewCmd(p *print.Printer) *cobra.Command {
 
 			projectLabel, err := projectname.GetProjectName(ctx, cmd, p)
 			if err != nil {
+				p.Debug(print.ErrorLevel, "get project name: %v", err)
```

Comment:
Not exactly sure about best practises here, but couldn't some of these logs be done in the function itself before returning the error instead of in all caller functions?

Reply:
also thought about this but was not sure if that would be okay. But I can change it

Reply:
I think that's breaking the error handling structure that we have, it would require some check before returning in the function which is not the standard.

Also, in that case we can't be sure that error won't be logged twice in the caller function

Reply:
In general, logging the error should happen on the most top-level possible. We are logging here because this error is being "silenced" by the CLI